File: API/GeorentateTrueAPI/ClassLibrary/BusinessEntities/AppInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ClassLibrary.BusinessEntities
{
    public class AppInfo
    {
        public string PrivacyPolicy { get; set; }
        public string About { get; set; }
        public string Conditions { get; set; }
        public string FBLink { get; set; }
        public string WebLink { get; set; }

        public AppInfo(string privacyPolicy, string about, string conditions, string fblink, string weblink)
        {
            PrivacyPolicy = privacyPolicy;
            About = about;
            Conditions = conditions;
            FBLink = fblink;
            WebLink = weblink;
        }

        public AppInfo()
        {
        }
    }
}

File: App/Georentate/app/src/main/java/com/example/zilvinastomkevicius/georentate/Entities/AppInfo.java
package com.example.zilvinastomkevicius.georentate.Entities;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonPropertyOrder;

public class AppInfo {

    @JsonProperty("PrivacyPolicy")
    public String PrivacyPolicy;
    @JsonProperty("About")
    public String About;
    @JsonProperty("Conditions")
    public String Conditions;
    @JsonProperty("Version")
    public String Version;
    @JsonProperty("FbLink")
    public String FBLink;
    @JsonProperty("WebLink")
    public String WebLink;
}
